### ISSUE TITLE
Integrate House Trackers with school plasma

### ISSUE THREAD

**Issue by Gum-Joe** [Gum-Joe/isitweeka#7]
Have the trackers from isitweeka displayed on the school plasma - this could work in many ways, for example, emailing a display slide every morning.

**Comment by ILikeTeaALot** [comment]
if a display slide being sent could be automated, that'd be great. otherwise.. I'll leave this to you

**Comment by Gum-Joe** [comment]
Yeah, I'll figure this out - see the issue desc. for what will be done. I do want this automated.
